java/struts2/src/main/resources/struts.xml

```diff
@@ -14,14 +14,10 @@
         <default-action-ref name="index"/>
         <global-allowed-methods>index,view,create</global-allowed-methods>
 
-        <action name="index">
-            <result>/WEB-INF/index.jsp</result>
-        </action>
+        <action name="index" class="org.apache.struts.benchmarker.example.UserAction"/>
 
         <action name="user/*" class="org.apache.struts.benchmarker.example.UserAction">
```

Comment:
Does this mean that `/user/a/4` is working ? I mean the _*_ does not mean any character allowed ?

Reply:
Yes, it's working but it will try to map `a/4` as a user ID

Reply:
In fact this project requires that `/user/1`, `/user/2` ... Works, but not any deeper

`/user/a/4` should not work

Reply:
Hm... need to double check and maybe fix that

Reply:
Let's fix this an other time